qa/L0_python_api/test_model_repository/identity/config.pbtxt

```diff
@@ -41,4 +41,10 @@ output [
     data_type: TYPE_STRING
     dims: [ 1 ]
   }
-]
\ No newline at end of file
+]
+instance_group [
+  {
+    count: 1
+    kind : KIND_CPU
```

Comment:
Did you ever investigate the gpu label thing?

Reply:
Investigated a bit, but did not find the root cause. Will create a ticket in my backlog with hopefully a more consistent reproducer.